app/tests/distributions/test_distributions_api.py

```diff
@@ -0,0 +1,727 @@
+def fixture_distribution_data(db, time_created):
+    provider = Provider.objects.create()
+    attribution_fields = {
+        "name_de": "BAFU",
+        "name_fr": "OFEV",
+        "name_en": "FOEN",
+        "name_it": "UFAM",
+        "name_rm": "UFAM",
+        "description_de": "Bundesamt für Umwelt",
+        "description_fr": "Office fédéral de l'environnement",
+        "description_en": "Federal Office for the Environment",
+        "description_it": "Ufficio federale dell'ambiente",
+        "description_rm": "Uffizi federal per l'ambient",
+        "provider": provider,
+    }
+    attribution = Attribution.objects.create(**attribution_fields)
+
+    dataset_fields = {
+        "slug": "ch.bafu.neophyten-haargurke",
+        "provider": provider,
+        "attribution": attribution,
+    }
+    with mock.patch('django.utils.timezone.now', mock.Mock(return_value=time_created)):
+        Dataset.objects.create(**dataset_fields)
+
+
+def test_attribution_to_response_returns_response_with_language_as_defined(distribution_data):
+
+    model = Attribution.objects.last()
+    actual = attribution_to_response(model, lang="de")
+
+    expected = AttributionSchema(
+        id=model.id,
+        name="BAFU",
+        name_translations=TranslationsSchema(
+            de="BAFU",
+            fr="OFEV",
+            en="FOEN",
+            it="UFAM",
+            rm="UFAM",
+        ),
+        description="Bundesamt für Umwelt",
+        description_translations=TranslationsSchema(
+            de="Bundesamt für Umwelt",
+            fr="Office fédéral de l'environnement",
+            en="Federal Office for the Environment",
+            it="Ufficio federale dell'ambiente",
+            rm="Uffizi federal per l'ambient",
+        ),
+        provider_id=Provider.objects.last().id
+    )
+
+    assert actual == expected
+
+
+def test_attribution_to_response_returns_response_with_default_language_if_undefined(
+    distribution_data
+):
+
+    model = Attribution.objects.last()
+    model.name_it = None
+    model.name_rm = None
+    model.description_it = None
+    model.description_rm = None
+
+    actual = attribution_to_response(model, lang="it")
+
+    expected = AttributionSchema(
+        id=model.id,
+        name="FOEN",
+        name_translations=TranslationsSchema(
+            de="BAFU",
+            fr="OFEV",
+            en="FOEN",
+            it=None,
+            rm=None,
+        ),
+        description="Federal Office for the Environment",
+        description_translations=TranslationsSchema(
+            de="Bundesamt für Umwelt",
+            fr="Office fédéral de l'environnement",
+            en="Federal Office for the Environment",
+            it=None,
+            rm=None,
+        ),
+        provider_id=Provider.objects.last().id,
+    )
+
+    assert actual == expected
+
+
+def test_get_attribution_returns_existing_attribution_with_default_language(
+    distribution_data, client, user_factory
```

Comment:
I'm wondering if we should be more concise with the names of the fixtures. `user_factory` is called "factory" but `distribution_data` is not. To me, `distribution_data` is also sort of a factory (even if we don't call it explicitly, it's sort of being called without argument). 
Maybe that's influenced from using https://pytest-factoryboy.readthedocs.io/en/stable/ in the past, idk. 
I'm not saying that we should now call them all `distribution_data_factory` per se, I just want to trigger some re-evaluating of the names. (This probably applies to most of the tests)

Reply:
OK now having seen all the tests maybe it's actually OK this way. I would consider renaming `user_factory` to make it clear that this is about the user that's connecting to the API, see my other comment.

Reply:
changed all data fixtures to individual object fixtures